draft-ietf-scitt-architecture.md

```diff
@@ -371,6 +371,156 @@ Issuers SHOULD use consistent decentralized identifiers for all their Statements
 They MAY update their DID manifest, for instance to refresh their signing keys or algorithms, but they SHOULD NOT remove or change any prior keys unless they intend to revoke all Signed Statements that are registered as Transparent Statements issued with those keys.
 This DID appears in the Issuer protected header of Signed Statements' Envelopes, while the version of the key from the manifest used to sign the Signed Statement is written in the `kid` header.
 
+`kid` MUST either be an absolute URL,
+or `iss` MUST also be present in the protected header,
+and `kid` MUST be a URL relative to `iss`.
+
+Resolving `kid` MUST return an Identity Document of a registered content type (a set of public keys).
+In the case of `kid` being an absolute DID URL, the identity document is called a DID Document,
+and is expected ot have content type `application/did+json`.
+
+Dereferencing a DID URL requires that it first be resolved, and then the fragment processed according to the media type.
+
+For example, when resolving `did:example:123#key-42`,
+first the identity document for `did:example:123` is resolved as content type `application/did+json`,
+next, the fragment `#key-2` is dereferenced to a verification method that contains a `publicKeyJwk` property in JSON.
+
+The content type of `publicKeyJwk` is expected to be `application/jwk+json`.
```

Comment:
why is there a need for a content type for a JWK? JWK is a published RFC and has a clearly defined content/structure

Reply:
Just being explicit.. and... because resolution and dereferencing are only described in the abstract in did core.

Most of this text would not be needed if DID Core defined resolution and dereferencing more concretely